sdk/zksend/src/wallet/index.ts

```diff
@@ -196,7 +196,7 @@ export class StashedWallet implements Wallet {
 					address,
 					chains: [SUI_MAINNET_CHAIN],
 					features: ['sui:signTransactionBlock', 'sui:signPersonalMessage'],
-					// NOTE: Stashed doesn't support getting public keys, and zkLogin accounts don't have meaningful public keys anyway
+
 					publicKey: new Uint8Array(),
```

Comment:
TODO: We need to update the stashed api to return the publicKey

Reply:
We need to the enoki SDK updates to land before we can implement this in stashed, so this PR will need to land without this first